SYCL/BFloat16/bfloat16_type.cpp

```diff
@@ -1,12 +1,13 @@
+// RUN: %if cuda %{%clangxx -fsycl -fsycl-targets=%sycl_triple -DUSE_CUDA_SM80=1 -Xsycl-target-backend --cuda-gpu-arch=sm_80 %s -o %t.out %}
+// RUN: %if cuda %{%GPU_RUN_PLACEHOLDER %t.out %}
```

Comment:
So there is an sm_80 in the CI now? Useful to know.

Reply:
I don't think there is, but with the check we just skip if the device does not support it. I'd argue it is generally a bad practice to write the tests in a way that makes them dependent on what we have in CI, as anyone running it on other systems would see failures if we do. You could argue though that skipping without more information than a hidden message is also a bad practice, but in my opinions it is the lesser evil. 😄

Reply:
I see. Yeah make sense thanks!